server/server.go

```diff
@@ -129,12 +129,11 @@ func (keys *DefaultKeystore) Get(op *protocol.Operation) (crypto.Signer, bool) {
 		priv, found := keys.skis[ski]
 		if found {
 			log.Infof("fetch key with SKI: %s", ski)
-			return priv, found
+			return priv, nil
 		}
```

Comment:
Wouldn't you do `return nil, nil` for case `!found`? If so, I'd also make the last error more explicit, that it's because SKI was invalid.